api-specs/api_spec_PGS.yaml

```diff
@@ -978,94 +1000,54 @@ components:
           type: string
           description: Expiration date of the card, yyyyMM format
           example: 203012
-    XPayPaymentRequest:
-      required:
-        - esito
-      type: object
-      properties:
-        esito:
-          type: string
-          enum: [ OK, KO ]
-          description: outhcome from xpay
-        idOperazione:
-          type: string
-          description: idOperazione from xpay
-          example: "123456"
-        xpayNonce:
-          type: string
-          description: nonce from xpay
-          example: "nonce"
-        timestamp:
-          type: string
-          description: timestamp in ms
-          example: 16589654852
-        mac:
-          type: string
-          description: mac from xpay
-          example: bd2dd7a959e4816b1ba9e767cf2a5287192c8be7
-        codice:
-          type: string
-          description: error code from xpay
-          example: 1
-        messaggio:
-          type: string
-          description: error message from xpay
-          example: error message from xpay
-    XPayAuthResponseEntity:
+    XPayPaymentAuthorizationRequestAccepted:       
       type: object
       required:
         - requestId
         - urlRedirect
         - timeStamp
-      properties:
-        requestId:
+       properties:
```

Comment:
It seems that you have to delete a blank before "properties" entry. Check It in swagger editor.

Reply:
thanks!